patterns/alz/policySetDefinitions/Deploy-LandingZone-Alerts.json

```diff
@@ -4176,6 +4176,74 @@
           "displayName": "FD Backend Request Latency Alert State",
           "description": "Alert state for the alert"
         }
+      },
```

Comment:
Please undo the modifications. No additional changes will be made to the Landing Zone Initiative.